packages/vscode-dependencies-validation/src/commands.ts

```diff
@@ -0,0 +1,40 @@
+import { dirname } from "path";
+import type { DiagnosticCollection } from "vscode";
+import { invokeNPMCommand } from "@sap-devx/npm-dependencies-validation";
+import { refreshDiagnostics } from "./diagnostics";
+import { VscodeOutputChannel } from "./vscodeTypes";
+
+export async function fixAllDepIssuesCommand(
+  outputChannel: VscodeOutputChannel,
+  packageJsonPath: string,
+  dependencyIssuesDiagnosticCollection: DiagnosticCollection
+): Promise<void> {
+  outputChannel.show(true);
```

Comment:
for automatic mode we should not always focus/show the output channel

Reply:
Yes, I know